test/e2e/commands/test_create_get_xmlupload.py

```diff
@@ -477,3 +449,53 @@ def _compare_resources(
                         msg=f"Onto '{onto_name}': Resource with original name '{orig['name']}' and returned name "
                         "'{ret['name']}' failed. The reason of the error lies OUTSIDE of the 'cardinalities' section.",
                     )
+
+    @staticmethod
```

Comment:
Also in this PR, I'm a little concerned about `@staticmethod` versus `@classmethod` versus neither of both